addons/mrp/migrations/15.0.2.1/pre-migration.py

```diff
@@ -0,0 +1,8 @@
+def migrate(cr, version):
+    cr.execute(
+        """
+        UPDATE mrp_production as mp
+        SET date_start = (SELECT MIN(date_start) FROM mrp_workorder WHERE production_id = mp.id) 
```

Comment:
Trailing whitespace should be removed
```suggestion
        SET date_start = (SELECT MIN(date_start) FROM mrp_workorder WHERE production_id = mp.id)
```

Reply:
@davidtranhp yes sir